client/soundland/src/Components/AboutUs.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class AboutUs {
+
+  render (){
+    return(
+        <div class="aboutus"><h2>About Team Yar-ss-ary-phanie</h2>
+            <div class="yarden"><h3>Yarden Hochman</h3>
+            <img src=""/>
+            <p></p>
```

Comment:
@Gnyc1001 maybe ad links to our github accounts. You can use the github icon as the link [icon here](https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwiq__-hpbDWAhUK2oMKHTF4CHUQjRwIBw&url=https%3A%2F%2Fwww.flaticon.com%2Ffree-icon%2Fgithub-logo_25231&psig=AFQjCNHOrcqCP6ecde-AAqkY953PXQNFVA&ust=1505877574731559 )